### PR TITLE
Wiring v26

### PR DESCRIPTION
**Pull request by MarcelMue**

### CODE REVIEW

File: service/controller/cluster.go [hunk 1]
```diff
@@ -28,7 +28,9 @@ import (
 	v24 "github.com/giantswarm/kvm-operator/service/controller/v24"
 	v24cloudconfig "github.com/giantswarm/kvm-operator/service/controller/v24/cloudconfig"
 	v25 "github.com/giantswarm/kvm-operator/service/controller/v25"
+	v26 "github.com/giantswarm/kvm-operator/service/controller/v25"
 	v25cloudconfig "github.com/giantswarm/kvm-operator/service/controller/v25/cloudconfig"
+	v26cloudconfig "github.com/giantswarm/kvm-operator/service/controller/v25/cloudconfig"
```
Comment (xh3b4sd) [inline service/controller/cluster.go]:
```suggestion
	v26cloudconfig "github.com/giantswarm/kvm-operator/service/controller/v26/cloudconfig"
```

File: service/controller/cluster.go [hunk 2]
```diff
@@ -28,7 +28,9 @@ import (
 	v24 "github.com/giantswarm/kvm-operator/service/controller/v24"
 	v24cloudconfig "github.com/giantswarm/kvm-operator/service/controller/v24/cloudconfig"
 	v25 "github.com/giantswarm/kvm-operator/service/controller/v25"
+	v26 "github.com/giantswarm/kvm-operator/service/controller/v25"
```
Comment (xh3b4sd) [inline service/controller/cluster.go]:
```suggestion
	v26 "github.com/giantswarm/kvm-operator/service/controller/v26"
```

File: service/version_bundles.go [hunk 3]
```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/giantswarm/kvm-operator/service/controller/v23patch1"
 	v24 "github.com/giantswarm/kvm-operator/service/controller/v24"
 	v25 "github.com/giantswarm/kvm-operator/service/controller/v25"
+	v25 "github.com/giantswarm/kvm-operator/service/controller/v26"
```
Comment (xh3b4sd) [inline service/version_bundles.go]: This should not even compile. 

```suggestion
	v26 "github.com/giantswarm/kvm-operator/service/controller/v26"
```

**Review by xh3b4sd — APPROVED** [review]
I think I passed the tests here.

**Comment by MarcelMue** [comment]
I am not a smart man. 
I think my brain just shut off when proof reading my own PR.

**Pull request merged**